package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "npm-merge-driver-install",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
This should stay `1.1.1`. I will bump the version to `1.1.2` when I do a release.